www/caddy/src/opnsense/mvc/app/models/OPNsense/Caddy/Caddy.php

```diff
@@ -59,7 +59,7 @@ private function checkForUniquePortCombos($items, $messages, $type = 'domain')
                 if (isset($combos[$comboKey])) {
                     // Use dynamic $key for message referencing
                     $messages->appendMessage(new Message(
-                        "Duplicate entry: The combination of $type '$fromDomainOrSubdomain' and port '$port' is already used. Each $type and port pairing must be unique.",
+                        sprintf(gettext("Duplicate entry: The combination of %1\$s '%2\$s' and port '%3\$s' is already used. Each %1\$s and port pairing must be unique."), $type, $fromDomainOrSubdomain, $port),
```

Comment:
What is "$type"? Do we really have to use it twice? It will be hard for translators to guess what "Each XXX" is.